.changeset/seven-donuts-yawn.md

```diff
@@ -0,0 +1,7 @@
+---
+"@pnpm/plugin-commands-installation": minor
+"@pnpm/sort-packages": minor
+"@pnpm-private/typings": minor
+---
+
+Check that graph sequencer result is safe
```

Comment:
this description seems not right for `@pnpm/sort-packages`. I suggest creating a separate changeset for it.

Reply:
Done, I changed the description and wrote some tests